packages/base/src/environment.json

```diff
@@ -209,6 +209,10 @@
       "name": "ClipboardEvent",
       "message": "This global is not available in the Node.js and browser environment."
     },
+    {
+      "name": "ClipboardItem",
```

Comment:
Were these changes also necessary to get things working on Windows or should these go in another PR?

Reply:
Yeah good question.  That's just what happened when I ran `yarn generate`, but maybe it's not necessary?

Reply:
It's possible these files are out of date, but yeah, if the tests still pass here without these changes I'd say leave them out and we can address them in a new PR.

Reply:
Hmm, so when I run `yarn generate`, I don't get anything. Which Node version are you using?

Reply:
16.20.0

Reply:
Ah - that might be the reason. The `.nvmrc` specifies v14, so that's the recommended development version of Node. Probably need to bump this, but in any case, I think that explains the difference in globals.